controllers/categoryController.js

```diff
@@ -0,0 +1,56 @@
+const _ = require("lodash");
+const product = require('../models').product;
+const product_categories = require('../models').product_categories;
+const categories = require('../models').categories;
+const { filter, prodAttributes, filteredProds } = require('./productsController');
+const { Op } = require("sequelize");
+
+
+prodByCategory = async (id, off, g, s, c) => {
```

Comment:
@Wajahat-Khan you did a great job. Suggesting something doesn't mean that you did it in wrong way. It's just to polish you skills and want get you at next level.

Let's use semantic names of variables. small instead of s, c - color.

Reply:
Okay, I will use the semantic names of variables. Thanks